sources/packages/web/src/views/student/StudentProfileEdit.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <student-page-container>
+    <template #header>
+      <header-navigator title="Student" subTitle="Profile" />
+    </template>
+    <student-profile-form
+      :processing="processing"
+      :formModel="initialData"
+      @submitted="submitted"
+      @customEvent="showPDApplicationModal"
+    />
+  </student-page-container>
+  <p-d-status-application-modal
+    max-width="600"
+    ref="pdStatusApplicationModal"
+  />
+</template>
+
+<script lang="ts">
+import { ref, onMounted } from "vue";
+import { useRouter } from "vue-router";
+import { ModalDialog, useSnackBar } from "@/composables";
+import {
+  StudentPDStatus,
+  StudentProfileFormModel,
+  StudentProfileFormModes,
+} from "@/types/contracts/StudentContract";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
+import PDStatusApplicationModal from "@/components/students/modals/PDStatusApplicationModal.vue";
+import { StudentService } from "@/services/StudentService";
+import { UpdateStudentAPIInDTO } from "@/services/http/dto/Student.dto";
+import { AuthService } from "@/services/AuthService";
+import { FormIOForm } from "@/types";
+import StudentProfileForm from "@/components/common/StudentProfileForm.vue";
+
+export default {
+  components: {
+    StudentProfileForm,
+    PDStatusApplicationModal,
+  },
+  setup() {
+    const router = useRouter();
+    const snackBar = useSnackBar();
+    const initialData = ref({} as StudentProfileFormModel);
+    const pdStatusApplicationModal = ref({} as ModalDialog<boolean>);
+    const processing = ref(false);
+    const showPendingStatus = ref(false);
```

Comment:
`showPendingStatus` is also not used in vue template

Reply:
Good point, I just checked the main branch and it was not used.
I removed it from here because it is received directly from the method `StudentService.shared.getStudentProfile()`. I just updated the form model to include the `pdStatus` also.
Good catch 👍